docs/guide/tutorial-4-more-about-controllers-20-minutes.md

```diff
@@ -946,16 +945,15 @@ while wb_robot_step(TIME_STEP) ~= -1
   left_speed  = 0.5 * MAX_SPEED;
   right_speed = 0.5 * MAX_SPEED;
   % modify speeds according to obstacles
-  if (left_obstacle) {
+  if (left_obstacle)
     % turn right
     left_speed  += 0.5 * MAX_SPEED;
     right_speed -= 0.5 * MAX_SPEED;
-  }
-  else if (right_obstacle) {
+  else if (right_obstacle)
```

Comment:
```suggestion
  elseif right_obstacle
```

In Matlab, `elseif` is concatenated: https://ch.mathworks.com/help/matlab/ref/if.html
I'm less sure, but the parenthesis is certainly useless here too.